sdk/eventhub/event-hubs/README.md

```diff
@@ -276,15 +276,21 @@ To control the number of events passed to processEvents, use the options argumen
 You can use `EventHubClient` to work with IotHub as well. This is useful for receiving telemetry data of IotHub from the linked EventHub.
 Most likely the associated connection string will not have send claims. Hence getting HubRuntimeInfo or PartitionRuntimeInfo and receiving events would be the possible operations.
 
-- Please notice that we are awaiting on the [createFromIotHubConnectionString](https://azure.github.io/azure-sdk-for-js/event-hubs/classes/eventhubclient.html#createfromiothubconnectionstring) method to get an instance of the EventHubClient. This is different from other static methods on the client. The method talks to the IotHub endpoint to get a redirect error which contains the EventHub endpoint to talk to. It then constructs the right EventHub connection string based on the information in the redirect error and returns an instance of the EventHubClient that you can play with.
+- Please notice that the connection string needs to contain the `Endpoint` key-value pair
+  in addition to the `SharedAccessKeyName` and `SharedAccessKey` key-value pairs.
+  It can optionally include the name of the IoT Hub as the `EntityPath` value, or the name can be passed separately to the
+  `EventHubClient` constructor.
+  e.g. "Endpoint=sb://my-iothub-namespace-<uid>.servicebus.windows.net/;SharedAccessKeyName=my-SA-name;SharedAccessKey=my-SA-key;EntityPath=my-iot-hub-name"
 
 ```javascript
-const client = await EventHubClient.createFromIotHubConnectionString("connectionString");
+const client = new EventHubClient("connectionStringWithEntityPath");
+// or
+const client = new EventHubClient("connectionString", "iot-hub-name");
 await client.getProperties();
 await client.getPartitionProperties("partitionId");
 ```
 
-**Notes:** For scalable and efficient receiving, please take a look at [azure-event-processor-host](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/eventhub/event-processor-host). The Event Processor host, internally uses the streaming receiver to receive events.
+**Notes:** For scalable and efficient receiving, please take a look at [EventProcessor](https://azure.github.io/azure-sdk-for-js/event-hubs/classes/eventprocessor.html). The EventProcessor, internally uses the batched receiver to receive events.
```

Comment:
Just curious - what makes this more scalable/efficient? I looked at the linked doc and it doesn't mention it.

Reply:
A better link would be https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-event-processor-host once the content there has been updated to be language agnostic

Reply:
Are you recommending that I wait to change the link until after the docs there have been updated? Right now those docs are specific to EPH, which we've updated significantly as EventProcessor.

Reply:
I recommend we wait until the docs there have been updated. Maybe log an issue so that we dont forget?

Reply:
Logged #5355 to track this work.